cli.js

```diff
@@ -41,6 +42,19 @@ const disableHooksParser = v => {
   }
 };
 
+const mapBaseUrlParser = v => {
+  // Example value for regular expression: https://schema.example.com/crm/:./test/docs/
+  // it splits on last occurrence of : into the groups all, url and folder
+  const re = /(.*):(.*)/g;
+  let mapping = [];
+  if ((mapping = re.exec(v))===null || mapping.length!==3) {
```

Comment:
just a comment that this statement will not work with such situation:
```bash
ag test/docs/apiwithref.json @asyncapi/html-template -o ./build/ --force-write --map-base-url https://schema.example.com/crm/
```

because `https://schema.example.com/crm/` has `:` so `const re = /(.*):(.*)/g;` returns 3 elements and therefor it contunes to work and fails when trying to resolve refs

```
Something went wrong:
Error: Error downloading https://schema.example.com/crm/shared.json 
getaddrinfo ENOTFOUND schema.example.com
    at dereference (/Users/wookiee/sources/generator/node_modules/@asyncapi/parser/lib/parser.js:155:11)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

Reply:
Thanks - good point! I fixed it. Of course it would be better to put cli.js under test.